spanner/batch.go

```diff
@@ -149,6 +149,9 @@ func (t *BatchReadOnlyTransaction) PartitionReadUsingIndexWithOptions(ctx contex
 			trace.TracePrintf(ctx, nil, "Error in recording GFE Latency. Try disabling and rerunning. Error: %v", err)
 		}
 	}
+	if metricErr := recordGFELatencyMetricsOT(ctx, md, "PartitionReadUsingIndexWithOptions", t.otConfig); metricErr != nil {
```

Comment:
With opencensus user had option to disable GFE latency metrics, after this change are we going to enable GFE latency always?

Reply:
Yes. For OpenTelemetry we have provided only one option spanner.EnableOpenTelemetryMetrics which will give both session and gfe metrics by default.